site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -37,116 +39,96 @@ The logical relationship between the Gateway API resources looks like this:
-and the application owners that allows them to be independent actors.
-Route-owners can specify that they will bind with all Gateways in the cluster,
-or only Gateways from a specific Namespace, with a specific label selector, or
-an individual Gateway. Similarly, Gateways provide the same level of control.
-This allows a cluster to be more self-governed, which requires less central
-administration to ensure that Routes are not over-exposed.
+and the application owners that allows them to be independent actors. Routes
+can only bind to specified gateways and Gateways provide a similar level of
+selection control over which Routes they will send traffic to by trusting
+specific namespaces in which those Routes live and operate. This allows a
+cluster to be more self-governed, which requires less central administration
+to ensure that Routes are not over-exposed.
 
 ## Resource Deployment
 
-The infrastructure team deploys the `shared-gateway` Gateway into the `infra-ns` 
-Namespace. 
+The infrastructure team deploys the `shared-gateway` Gateway into the `infra-ns`
+Namespace.
 
 ```yaml
-{% include 'v1alpha1/cross-namespace-routing/gateway.yaml' %}
+{% include 'v1alpha2/cross-namespace-routing/gateway.yaml' %}
 ```
 
 A few notes about this Gateway:
 
-- It is matching for the `foo.example.com` domain. This is configured on the 
-Gateway so that each HTTPRoute does not also have to configure hostname matching, 
-since they are all using the same domain. This also allows these HTTPRoute 
-manifests to be reused across production and dev environments where the dev 
+- It is matching for the `foo.example.com` domain. This is configured on the
+Gateway so that each HTTPRoute does not also have to configure hostname matching,
+since they are all using the same domain. This also allows these HTTPRoute
+manifests to be reused across production and dev environments where the dev
 environment might be hosted at `foo.dev.corp.example.com`.
-- The Gateway is configured for HTTPS and references the `foo-example-com` Secret. 
-This allows the certificate to be managed centrally for all applications which 
+- The Gateway is configured for HTTPS and references the `foo-example-com` Secret.
+This allows the certificate to be managed centrally for all applications which
 are using this Gateway.
-- It allows any Route in the cluster to use this Gateway because `namespaces.from = All`. 
-This is a permissive method of Route selection since the Routes are given 
-full control to select this Gateway. There are more restrictive forms of Route 
-selection that allow selection on a per-Namespace basis, detailed 
+- It allows any Route in the cluster to use this Gateway because `namespaces.from = All`.
+This is a permissive method of Route selection since the Routes are given
+full control to select this Gateway. There are more restrictive forms of Route
+selection that allow selection on a per-Namespace basis, detailed
```

Comment:
I think we should remove all references to Route selection, maybe remove these lines?

Reply:
done in 890861739ef42a42882a1241b3ae005cd833c301